scripts/minishift-all-tests.sh

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env bash
+
+executing() {
+   set +x
+  echo -e "\n------------------------------\n${1}\n------------------------------\n"
+   set -x
+}
+
+set -ex
+
+#Export GitHub token to avoid 
+export MINISHIFT_GITHUB_API_TOKEN="03d362b0f2a7ecf02b30bfbdd40fe720a4d6b8b3"
+executing "Setting up environment..."
+
+export PATH="$PATH:/usr/local/go/bin/"
+export GOPATH=$HOME/go
+git clone https://github.com/openshift/odo.git openshift/odo
```

Comment:
This is cloning the master branch. Shouldn't it clone PR branch?
Also isn't this take care of by CI itself?

Reply:
Yes you don't have to clone. ci-firewall will already clone repo and check out the pr

Reply:
removing the cloning